hack/.deployUtils

```diff
@@ -81,7 +27,108 @@ metadata:
 EOF
 }
 
+deployOLM(){
+  clusterName=${1}
+
+  kubectl config use-context kind-${clusterName}
+  echo "Installing OLM in ${clusterName}"
+
+  ${OPERATOR_SDK_BIN} olm install --timeout 6m0s
+}
+
+deployOCMHub(){
+  clusterName=${1}
+  minimal=${2}
+  echo "installing the hub cluster in kind-(${clusterName}) "
+
+  kubectl config use-context kind-${clusterName}
+  ${CLUSTERADM_BIN} init --bundle-version='0.11.0' --wait --context kind-${clusterName}
+  echo "PATCHING CLUSTERMANAGER: placement image patch to use amd64 image - See https://kubernetes.slack.com/archives/C01GE7YSUUF/p1685016272443249"
+  kubectl patch clustermanager cluster-manager --type='merge' -p '{"spec":{"placementImagePullSpec":"quay.io/open-cluster-management/placement:v0.11.0-amd64"}}'
+  echo "checking if cluster is single or multi"
+  if [[ -n "${OCM_SINGLE}" ]]; then
+    clusterName=kind-${KIND_CLUSTER_CONTROL_PLANE}
+    echo "Found single cluster installing hub and spoke on the one cluster (${clusterName})"
+    join=$(${CLUSTERADM_BIN} get token --context ${clusterName} |  grep -o  'clusteradm.*--cluster-name')
+    ${BIN_DIR}${join} ${clusterName} --bundle-version='0.11.0' --feature-gates=RawFeedbackJsonString=true --force-internal-endpoint-lookup --context ${clusterName} | grep clusteradm
+    echo "accepting OCM spoke cluster invite"
+
+    max_retry=18
+    counter=0
+    until ${CLUSTERADM_BIN} accept --clusters ${clusterName}
+    do
+      sleep 10
+      [[ counter -eq $max_retry ]] && echo "Failed!" && exit 1
+      echo "Trying again. Try #$counter"
+      ((++counter))
+    done
+    deployOLM ${KIND_CLUSTER_CONTROL_PLANE}
```

Comment:
Since this was existing behaviour we do not need to look into this as part of this PR, but we should make sure we have follow up issues to look into this.